src/cf-base.less

```diff
@@ -536,26 +506,20 @@ table {
 
 th,
 td {
-    padding: unit(12px / @base-font-size-px, em) unit(15px / @base-font-size-px, em);
-    background: @td-bg;
+    padding: 10px;
 
     thead & {
         color: @thead-text;
         background: @thead-bg;
-    }
-
-    tbody > tr:nth-child(odd) > & {
-        background: @td-bg-alt;
-    }
-
-    .compact-table & {
-        padding: unit(7px / @base-font-size-px, em) unit(10px / @base-font-size-px, em);
+        font-size: .875em;
+        text-transform: uppercase;
+        .webfont-demi();
```

Comment:
Good catch!

Honest question, though - In the category of 'unexpected consequences,' though, should we worry about the "H5" style changing without the "THEAD TR" style changing? The design specification doesn't mention that the THEAD should match "H5" style, specifically.

Or am I being overly paranoid about style changes? :smile: :dancer: :rocket: